src/main/java/cloud/fogbow/ras/core/plugins/interoperability/opennebula/compute/v5_4/OpenNebulaComputePlugin.java

```diff
@@ -123,14 +120,14 @@ protected ComputeInstance doGetInstance(VirtualMachine virtualMachine) {
 		String state = virtualMachine.lcmStateStr();
 
 		int cpu = Integer.parseInt(virtualMachine.xpath(TEMPLATE_CPU_PATH));
-		int memory = Integer.parseInt(virtualMachine.xpath(TEMPLATE_MEMORY_PATH));
+		int memoryRam = Integer.parseInt(virtualMachine.xpath(TEMPLATE_MEMORY_PATH));
```

Comment:
I have left "memoryRam" because the name, in the Opennebula case, is MEMORY and the name, in the Fogbow case, is just RAM. There is a conflict so I have put "memoryRam" only in this situation.